### ISSUE TITLE
[bug]: Model installation doesn't proceed after download finishes

### ISSUE THREAD

**Issue by psychedelicious** [invoke-ai/InvokeAI#5807]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Linux

### GPU vendor

Nvidia (CUDA)

### Version number

next

### Browser

FF

### What happened

When I initiate a model install via the API using a HuggingFace URL or repo id, the model is downloaded but the installation never happens. I tried 2x safetensors and a diffusers. Here's my `invokeai/models/` after:

```sh
❯ tree tmp* -h
[128K]  tmpinstall_b0tgnn6y
└── [2.0G]  Reliberate_v3.safetensors
[128K]  tmpinstall_omd5n2xh
└── [2.0G]  Reliberate_v2.safetensors
[128K]  tmpinstall_wnoelcl9
└── [128K]  juggernaut-xl-v5
    ├── [ 577]  model_index.json
    ├── [128K]  scheduler
    │   └── [ 474]  scheduler_config.json
    ├── [128K]  text_encoder
    │   ├── [ 560]  config.json
    │   ├── [235M]  model.safetensors
    │   └── [235M]  pytorch_model.bin
    ├── [128K]  text_encoder_2
    │   ├── [ 570]  config.json
    │   ├── [1.3G]  model.safetensors
    │   └── [1.3G]  pytorch_model.bin
    ├── [128K]  tokenizer
    │   ├── [512K]  merges.txt
    │   ├── [ 472]  special_tokens_map.json
    │   ├── [ 737]  tokenizer_config.json
    │   └── [1.0M]  vocab.json
    ├── [128K]  tokenizer_2
    │   ├── [512K]  merges.txt
    │   ├── [ 460]  special_tokens_map.json
    │   ├── [ 725]  tokenizer_config.json
    │   └── [1.0M]  vocab.json
    ├── [128K]  unet
    │   ├── [1.7K]  config.json
    │   └── [4.8G]  diffusion_pytorch_model.safetensors
    └── [128K]  vae
        ├── [ 602]  config.json
        └── [160M]  diffusion_pytorch_model.safetensors
```

Installing from a local path works

### What you expected to happen

I can download models from HF

**Comment by psychedelicious** [comment]
I'm having trouble wrapping my head around the application flow and figuring out what is going wrong here.

**Comment by psychedelicious** [comment]
Resolved by #5835
